kankaview/kankaview.py

```diff
@@ -469,12 +475,17 @@ def __init__(self):
         em.add_field(name='Created At',
                      value=campaign.created_at[:-11].replace('T', ' '))
         em.add_field(name='Updated At',
-                     value=campaign.updated_at[:-11].replace('T', ' '))
+                     value=campaign.created_at[:-11].replace('T', ' '))
```

Comment:
Shouldn't this be `updated_at`?
```suggestion
                     value=campaign.updated_at[:-11].replace('T', ' '))
```

Reply:
Yep